.changeset/changelog-lit.cjs

```diff
@@ -101,16 +99,44 @@ const changelogFunctions = {
 
     const users = usersFromSummary.length
       ? usersFromSummary
+          .filter((u) => !containsLitTeamMemberUsername(u))
```

Comment:
This removes any mention of the user, yeah? I don't think that's what we want. I think we just want to say "Thanks" only to the external contributors.

Reply:
line 139 below has the `users` check that conditionally adds the whole "Thanks" phrase with the users' github handle linked.
```ts
    const prefix = [
      links.pull === null ? '' : ` ${links.pull}`,
      links.commit === null ? '' : ` ${links.commit}`,
      users === null ? '' : ` Thanks ${users}!`,
    ].join('');
```
If user is a lit team member, `users` would be null.

Or are you saying you want to just say "Thanks" and not have the user's github handle linked?

Reply:
Ah this is a case that our current changesets are not covering. In this code path that isn't exercised the result of filtering out team members would result in an empty string. An empty string would fail to match with `null` and would print an empty `Thanks`.

I will make the null handling also check for an empty string & locally test this code path.

Reply:
Fixed by also checking for an empty string & manually tested the cases with multiple `author:<author>` written in a changeset file.

Reply:
Understood! Although I misunderstood you, you caught a real bug that could have given unattributed thanks! 😄 

Current behavior is that all changes ignore users & have no attribution or thanks.

This PR: Adds attribution & thanks to external contributors, but core members continue to receive no thanks or attribution.

Your proposal is to always add attribution, but only add the additional prefixed "Thanks <contributor list>!" if it's an external contributor?
Would you like me to change to always include attribution?

Reply:
Ooh I totally misunderstood that. I'm less sure if we need the attribution for team members since clicking on the PR link right before it would show who worked on it. But no strong feelings either way.